### PR TITLE
Leader needs to bounce not just a Master

### PR DESCRIPTION
**Pull request by kensipe**
Leader needs to bounce not just a Master

Summary:
This was originally built and run with 1 master clusters.  This now is what is intended for 1, 3 or 5 master clusters.

https://jira.mesosphere.com/browse/DCOS-40567

JIRA issues:  DCOS-40567

### CODE REVIEW

File: tests/system/test_root_metronome.py [hunk 1]
```diff
@@ -124,8 +124,9 @@ def test_disable_schedule_recovery_from_master_bounce():
         job_schedule['enabled'] = False
         client.update_schedule(job_id, 'nightly', job_schedule)
 
-        # # bounce master
-        shakedown.restart_master_node()
+        # bounce master
+        metronome_leader = shakedown.master_leader_ip()
```
Comment (alenkacz) [inline tests/system/test_root_metronome.py]: unfortunately mesos leader != metronome leader. This does not have to be true. This is why I linked the ticket to the other ticket that is adding metronome leader info into the API...

Comment (kensipe) [reply tests/system/test_root_metronome.py]: ah... that is what you mean :) .  great pt!   good eye!  and thanks.

File: tests/system/test_root_metronome.py [hunk 2]
```diff
@@ -281,8 +281,7 @@ def test_metronome_shutdown_with_no_extra_tasks():
         # restart metronome process
```
Comment (alenkacz) [inline tests/system/test_root_metronome.py]: we should drop this comment

Comment (kensipe) [reply tests/system/test_root_metronome.py]: to avoid conflict and duplication I rebased this on the other PR... removing.

**Pull request merged**